frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -31,66 +29,36 @@ import { DiffModal } from '../../components/DiffModal'
 export function PolicyTemplateDetails(
   props: Readonly<{
     clusterName: string
-    apiGroup: string
-    apiVersion: string
-    kind: string
-    templateName: string
-    setParentTemplate: Dispatch<any>
+    template: any
```

Comment:
It almost feels like `template` should be a type that contains the props that were removed, but that might be a refactor for another day (if it's valid).

Reply:
It's a good point but I'd rather leave that out for expediency of getting the bugs fixed with RC quickly approaching.